src/index.html

```diff
@@ -20,8 +24,364 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="page__content">
+    <header class="page__header header">
+      <a
+        class="icon"
+        href="#logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
```

Comment:
Consider providing a more descriptive alt text for the logo image to improve accessibility. Instead of 'logo', use something like 'Company Logo'.